Example/common.js

```diff
@@ -35,3 +33,7 @@ Praesent ac risus nec augue commodo semper eu eget quam. Donec aliquam sodales c
 
 Fusce interdum nisi a risus aliquet, non dictum metus cursus. Praesent imperdiet sapien orci, quis sodales metus aliquet id. Aliquam convallis pharetra erat. Fusce gravida diam ut tellus elementum sodales. Fusce varius congue neque, quis laoreet sapien blandit vestibulum. Donec congue libero sapien, nec varius risus viverra ut. Quisque eu maximus magna. Phasellus tortor nisi, tincidunt vitae dignissim nec, interdum vel mi. Ut accumsan urna finibus posuere mattis.
 `;
+
+export const alert = message => {
+  window.alert(message);
```

Comment:
Is the native apps use window.alert too?

I think it's better just to use global `alert`.

Reply:
Okay, it's in now.